### PR TITLE
[FIX] Harass system overly harassing bases

### PR DESCRIPTION
**Pull request by dijksterhuis**
I've just copied things into new files to make the logic separation easier for others. Also means we can easily turn this off and on by editing one file `server/functions/harass/fn_harass_job.sqf`

90% of the original code is exactly the same, I'll highlight main logic change in comments

---

FOBs had attack AI objectives assigned to them **and** pursuit AI objectives assigned to them.

Players in a FOB are not moving around anywhere .... so why do they need to be pursued by tracker teams, who have specific behaviour logic related to following players and giving up the objective if they lose the target?

TL;DR -- variables seem to have been set in the wrong order, meaning we were getting ~60 pursue (tracker team) objectives assigned to FOBs ... when we should have just had a series of attack objectives assigned.

### TODOs

- [ ] Needs testing to verify the syntax (written on my linux machine with no copy of Arma)
- [ ] Needs testing to verify logic change (no pursuit AI groups should be assigned when player is in a base area)

### CODE REVIEW

File: server/functions/harass/fn_harass_create_squads_v2.sqf [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	private _playersHarrassedBaseAttacks = [_harassablePlayers] call para_s_fnc_harass_attack_base;
+
+	// Find players that aren't
+	// - harassed enough by tracker teams
+	// - being harrased by base attacks
+
+	private _friendlyPlayersToHarass = _harassablePlayers select {_x getVariable "harass_level" < 1};
+	private _playersToPursueHarrass = _friendlyPlayersToHarass - _playersHarrassedBaseAttacks;
+
+	private _playersHarrassedPursuits = [_playersToPursueHarrass] call para_s_fnc_harass_pursue_players;
```
Comment (dijksterhuis) [inline server/functions/harass/fn_harass_create_squads_v2.sqf:82]: This is where the main logic has changed -- patched version explicitly disables tracker teams / pursuit AI from being added to players that are affected by the logic to harass bases

**Comment by dijksterhuis** [comment]
closing, this is a bit overboard for changing location of one line.
